redux_userlist_06/src/actions/actions.ts

```diff
@@ -0,0 +1,30 @@
+import { UserDetails } from "../interfaces";
+import * as constants from "../constants/constants"
+export const insertUser = (payload:UserDetails)=>{
+    console.log(payload)
+    return {
+        type:constants.CREATE,
+        payload:payload
+    }
```

Comment:
remove the logs, and use can avoid writing return keyword if there is one line code to return:
e.g export const addUserList = (payload:UserDetails[])=>( {
        type:constants.INSERTALL,
        payload:payload
    });